packages/react-native/sdks/hermes-engine/utils/replace_hermes_version.js

```diff
@@ -53,7 +53,7 @@ function shouldReplaceHermesConfiguration(configuration) {
 }
 
 function replaceHermesConfiguration(configuration, version, podsRoot) {
-  const tarballURLPath = `${podsRoot}/hermes-engine-artifacts/hermes-ios-${version}-${configuration}.tar.gz`;
+  const tarballURLPath = `${podsRoot}/hermes-engine-artifacts/hermes-ios-${version.toLowerCase()}-${configuration.toLowerCase()}.tar.gz`;
```

Comment:
I don't think we have to lower case the version, or am I missing something?

Reply:
In stable versions it is really not necessary but I added it for 2 reasons:
1. Because of the .RC versions as you said and I saw in the issue: https://github.com/facebook/react-native/issues/39903 some people having this same error and it could be because of this issue.
2. The method that generates the files in `hermes_utils.js` reduces the entire value including the version. So I think it really affects RC versions.